### PR TITLE
Organize the README.

### PR DESCRIPTION
**Pull request by cdrage**
This updates the README with some small changes such as titles as well
as rewording a few sentences.

### CODE REVIEW

**Comment by surajssd** [comment]
@cdrage only a minor comment otherwise LGTM

File: README.md [hunk 1]
```diff
@@ -90,25 +91,19 @@ $ make binary-cross
...
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
-
-You can reach the maintainers of this project at:
+## Community, Discussion, Contribution, and Support
 
-- [Slack](http://slack.kubernetes.io): #kompose
+As part of the Kubernetes ecosystem, we follow the Kubernetes community principles. More information can be found on the [community page](http://kubernetes.io/community/).
 
-Incubation of `kompose` into the Kubernetes project is shepherd in the [SIG-APPS](https://github.com/kubernetes/community).
+You can reach the maintainers of this project on [Slack](http://slack.kubernetes.io) in channel #kompose
```
Comment (cdrage) [inline README.md]: if we put contributors, we'd have to put `You can reach some of the contributors`. When you put `reach` it implies someone will always be there, in our case the maintainers will more then likely always be there :)

Comment (surajssd) [inline README.md]: My only concern is that it should not sound like "not welcoming for everyone to hangout on #kompose channel".
Whatever does not sound like above is good for me.

File: README.md [hunk 2]
```diff
@@ -90,25 +91,19 @@ $ make binary-cross
...
 
-[@ericchiang](https://github.com/ericchiang) is acting as Champion for [incubation](https://github.com/kubernetes/community/blob/master/incubator.md)
+`kompose` is being incubated into the Kubernetes community via [SIG-APPS](https://github.com/kubernetes/community/tree/master/sig-apps) on [kubernetes/community](https://github.com/kubernetes/community).
 
-We do a bi-weekly community meeting. Here is the link to [agenda doc](https://docs.google.com/document/d/1I5I21Cp_JZ9Az5MgMcu6Hl7m8WQ1Eqk_WeQLHenNom0/edit?usp=sharing).
+[@ericchiang](https://github.com/ericchiang) is acting champion for [incubation](https://github.com/kubernetes/community/blob/master/incubator.md).
 
-Meeting link: [https://bluejeans.com/404059616](https://bluejeans.com/404059616)
+We do a biweekly community meeting which is [open to the public](https://bluejeans.com/404059616). Each week we outline what we have talked about in an [agenda doc](https://docs.google.com/document/d/1I5I21Cp_JZ9Az5MgMcu6Hl7m8WQ1Eqk_WeQLHenNom0/edit?usp=sharing).
```
Comment (ericchiang) [inline README.md]: What time this meeting at?

Comment (cdrage) [inline README.md]: May change each week, so it's in the Google doc. I'll update this PR with that change.

**Comment by cdrage** [comment]
@kadel @ericchiang rebased ^^

File: README.md [hunk 3]
```diff
@@ -95,25 +97,19 @@ $ make binary-cross
...
 
-[@ericchiang](https://github.com/ericchiang) is acting as Champion for [incubation](https://github.com/kubernetes/community/blob/master/incubator.md)
+`kompose` is being incubated into the Kubernetes community via [SIG-APPS](https://github.com/kubernetes/community/tree/master/sig-apps) on [kubernetes/community](https://github.com/kubernetes/community).
 
-We do a bi-weekly community meeting. Here is the link to [agenda doc](https://docs.google.com/document/d/1I5I21Cp_JZ9Az5MgMcu6Hl7m8WQ1Eqk_WeQLHenNom0/edit?usp=sharing).
+[@ericchiang](https://github.com/ericchiang) is acting champion for [incubation](https://github.com/kubernetes/community/blob/master/incubator.md).
 
-Meeting link: [https://bluejeans.com/404059616](https://bluejeans.com/404059616)
+We do a biweekly community meeting which is [open to the public](https://bluejeans.com/404059616). Each week we outline what we have talked about in an [agenda doc](https://docs.google.com/document/d/1I5I21Cp_JZ9Az5MgMcu6Hl7m8WQ1Eqk_WeQLHenNom0/edit?usp=sharing). The time and date of the meeting can be looked up on the Google doc (subject to change).
```
Comment (kadel) [inline README.md]: Can we put actual  time and date here?
We can always update it if it changes.

Comment (cdrage) [inline README.md]: Sure. I've added it in.

**Pull request merged**